test/workos/sso/sso_test.exs

```diff
@@ -13,7 +13,12 @@ defmodule WorkOS.SSOTest do
     setup do
       {:ok, url} =
         SSO.get_authorization_url(
-          %{domain: "test", provider: "GoogleOAuth", redirect_uri: "project", state: "nope"},
+          %{
+            organization: "test",
+            provider: "GoogleOAuth",
```

Comment:
`organization` and `provider` are actually mutually-exclusive options, so for this test could we just keep `organization` and remove `provider`?